src/pages/Material/index.js

```diff
@@ -1,5 +1,34 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import { compareDesc } from 'date-fns';
+import { Main, Materials } from './style';
+import Card from '../../components/Card';
+import ActivityHistory from '../../components/ActivityHistory';
 
-const Material = () => <h1>Material Page</h1>;
+const Material = () => {
+  const materials = useSelector((state) => state.materials);
+  const loans = useSelector((state) =>
+    state.loans.slice().sort((a, b) => compareDesc(a.updated_at, b.updated_at))
+  );
```

Comment:
As said before, instead of store all the loans into a state, we just fetch them and then we doing something like this.